test/vtctld_test.py

```diff
@@ -228,6 +230,18 @@ def test_tablet_status(self):
     self.assertIn('Alias: <a href="http://localhost:', shard_0_replica_status)
     self.assertIn('</html>', shard_0_replica_status)
 
+  def test_interrupt_vtctl_command(self):
+    """An interrupted streaming vtctl command should work."""
+    protocol, endpoint = utils.vtctld.rpc_endpoint(python=True)
+    vtctld_connection = vtctl_client.connect(protocol, endpoint, 30)
+    i = 0
+    for event in vtctld_connection.execute_vtctl_command(['ListAllTablets',
```

Comment:
This is fine :smile: . But just for reference, the more Pythonic version:

for i, event in enumerate(events):